### PR TITLE
translations/cs: Add Czech translation

### PR DESCRIPTION
**Pull request by TenTeypek**
I hope I haven't forgotten anything.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ i18next
     react: {
       useSuspense: true
     },
-    supportedLngs: ['de', 'en', 'es', 'fr', 'nl', 'pl', 'pt', 'ru', 'tr', 'hu', 'it', 'ml']
+    supportedLngs: ['de', 'en', 'es', 'fr', 'nl', 'pl', 'pt', 'ru', 'tr', 'hu', 'it', 'ml','cs']
```
Comment (adityaruplaha) [inline src/index.tsx:36]: Just for consistency, please add a space after the comma.

File: i18next-parser.config.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ module.exports = {
   lineEnding: 'auto',
   // Control the line ending. See options at https://github.com/ryanve/eol
 
-  locales: ['de', 'en', 'es', 'fr', 'nl', 'pl', 'pt', 'ru', 'tr', 'hu', 'it', 'ml'],
+  locales: ['de', 'en', 'es', 'fr', 'nl', 'pl', 'pt', 'ru', 'tr', 'hu', 'it', 'ml','cs'],
```
Comment (adityaruplaha) [inline i18next-parser.config.js:38]: Just for consistency, please add a space after the comma.

**Comment by adityaruplaha** [comment]
Also please fix the build and lint errors.

**Comment by TenTeypek** [comment]
Sorry, I am new to this stuff. I hope everything is alright now!

**Comment by adityaruplaha** [comment]
After #312 gets merged, I'll give this a quick test, and we are ready to merge then.

**Review by adityaruplaha — APPROVED** [review]
Tested. Should be fine. Merging.

**Pull request merged**